twitterscraper/tweet.py

```diff
@@ -4,9 +4,9 @@
 from coala_utils.decorators import generate_ordering
 
 
-@generate_ordering('timestamp', 'id', 'text', 'user', 'replies', 'retweets', 'likes')
+@generate_ordering('timestamp', 'id', 'text', 'user', 'replies', 'reply_to_id', 'retweets', 'likes')
 class Tweet:
-    def __init__(self, user, fullname, id, url, timestamp, text, replies, retweets, likes, html):
+    def __init__(self, user, fullname, id, url, timestamp, text, reply_to_id, replies, retweets, likes, html):
```

Comment:
placing a new argument at this location breaks backward compatibility. I suggest you move it to the end of the list of arguments.

Reply:
The newly implemented 'reply_to_user' is not passed to the Tweet class and hence will not appear in the output.